src/models/Config.ts

```diff
@@ -53,6 +53,7 @@ export const baseConfig = {
         pairedDataElement: "DM_PAIRED_DE",
         extraDataElement: "DM_EXTRA_INFO_DE",
         createdByApp: "DM_CREATED_BY_DATA_MANAGEMENT",
+        lastUpdatedData: "DM_LAST_UPDATED_DATA",
```

Comment:
We need this attribute to be automatically created. We have the infrastructure, "migrations", for this. Check folder `src/migrations/tasks/`, you'll have to create a new task (and add it to the list). Check the existing tasks to get the idea, for example `src/migrations/tasks/04.set-livelihoods-code.ts`.

Migrations are automatically checked/run when the app is started (by an admin user).

Reply:
i'm not entirely sure how to write the migration task for this. i've done what i think is correct. but i'm not sure.